src/overview-dashboard/sqs-dlq-stats-widget-function.lambda.ts

```diff
@@ -0,0 +1,204 @@
+/// @singleton SQSDLQStatsWidget-Handler
+const DOCS = `
+## Include Text Widget from CloudWatch Dashboard
+This widget displays the first text widget from a specified CloudWatch Dashboard. 
+
+This is useful for embedding the same text context in multiple dashboards and update it from a central place. An example would be a menu of links between dashboards.
+
+### Widget parameters
+Param | Description
+---|---
+**title** | Title of the widget
+**emptyQueueMessage** | Message to display when there are no visible messages in any of the queues
+**nonEmptyQueueMessage** | Message to display when any of the queue contains a message
+**queues** | Object containing information about the queues to display
+
+### Example parameters
+\`\`\` yaml
+{
+    "description": "",
+    "title": "SQS DLQ",
+    "emptyQueueMessage": "There are no messages in the DLQs. This is normal and no action is required.",
+    "nonEmptyQueueMessage": "There are some message in the DLQ. The table below lists the queues with non-empty DLQs. Please check the DLQs and re-drive the messages.",
+    "queues": {
+        "construct-hub-dev-ConstructHubOrchestrationDLQ9C6D9BD4-rlooG5g8o3YA": {
+            "name": "Orchestration DLQ",
+            "queueName": "construct-hub-dev-ConstructHubOrchestrationDLQ9C6D9BD4-rlooG5g8o3YA",
+            "reDriveFunctionArn": "arn:aws:lambda:us-east-2:123457424:function:construct-hub-dev-ConstructHubOrchestrationRedrive-bWYSY5KWfUUU"
+        },
+        "construct-hub-dev-ConstructHubIngestionDLQ3E96A5F2-6niMeQlEH0X0": {
+            "name": "Ingestion DLQ",
+            "queueName": "construct-hub-dev-ConstructHubIngestionDLQ3E96A5F2-6niMeQlEH0X0"
+        },
+        "construct-hub-dev-ConstructHubSourcesStagerDLQ80BD2600-AlCVN3KCXFPo": {
+            "name": "NPM JS Stager DLQ",
+            "queueName": "construct-hub-dev-ConstructHubSourcesStagerDLQ80BD2600-AlCVN3KCXFPo"
+        }
+    }
+}
+\`\`\`
+`;
+
+import { CloudWatch } from 'aws-sdk';
+const DURATION = 5; // minutes
+
+interface Event {
+  readonly key: string;
+  readonly description: string;
+  readonly widgetContext: WidgetContext;
+  describe?: unknown;
+}
+
+type QueueConfig = {
+  queueName: string;
+  name: string;
+  reDriveFunctionArn?: string;
+}
+
+export async function handler(event: Event): Promise<string | { markdown: string }> {
+  console.log(event);
```

Comment:
I think this can help prevent the object from getting truncated if the `event` is very big for some reason. Just a good practice
```suggestion
  console.log(JSON.stringify(event));
```